src/django/contricleaner/lib/dto/list_dto.py

```diff
@@ -0,0 +1,7 @@
+from typing import List, NamedTuple
+from contricleaner.lib.dto.row_dto import RowDTO
+
+
+class ListDTO(NamedTuple):
+    rows: List[RowDTO]
```

Comment:
Let's set the default value to an empty array `[]`. It will be useful in cases where you need to check whether it contains any values or not.

Reply:
the same for errors